dts/arm/st/l4/stm32l4p5.dtsi

```diff
@@ -12,6 +12,7 @@
 
 / {
 	sram0: memory@20000000 {
+		compatible = "mmio-sram";
```

Comment:
This isn't the correct fix.
The fix is to remove `/delete-node/ &sram0;`. This would follow the scheme used on other series.
Introduced by https://github.com/zephyrproject-rtos/zephyr/pull/55028

Reply:
Done. PTAL at this PR.